### ISSUE TITLE
Project Manager not showing up no matter what link you click in the navigation

### ISSUE THREAD

**Issue by ibanxx** [weDevsOfficial/wp-project-manager#454]
Plugin installation - no issues so seen
Admin navigation shows up.
No content showing up when u klick on either one or another link in the navigation.
So no settings, project, task etc. can be managed.

Did someone face the same issue?
WP ERP/CRM installed at the same time within this plugin, and those seem to work fine so far.

Thanks

**Comment by usman244718** [comment]
Yes, I am available. Please share some details about the task. I am eager to collaborate with you. with 6+ years experience, please dm me if you want my portfolio and resume. #fullstack development, #graphic designing
